Packs/AzureRiskyUsers/Integrations/AzureRiskyUsers/AzureRiskyUsers.py

```diff
@@ -96,16 +96,21 @@ def get_token_retrieval_url_by_auth_type(authentication_type: str) -> None | str
         else:  # Device Code Flow
             return 'https://login.microsoftonline.com/organizations/oauth2/v2.0/token'
 
-    def risky_users_list_request(self, limit: int = None, risk_state: str | None = None,
+    def risky_users_list_request(self, limit: int = None,
+                                 risk_state: str | None = None,
                                  risk_level: str | None = None,
-                                 skip_token: str | None = None) -> dict:
+                                 skip_token: str | None = None,
+                                 order_by: str | None = None,
+                                 update_before: str | None = None,
+                                 updated_after: str | None = None) -> dict:
         """
         List risky users.
 
         Args:
             risk_state (str): Risk State to retrieve.
             risk_level (str): Specify to get only results with the same Risk Level.
             limit (int): Limit of results to retrieve.
+            order_by (str): Order results by this attribute.
```

Comment:
Please add the rest of the arguments to the doc string as well (update_before, updated_after). Here and in the rest of the functions.

Reply:
Done updated the integration accordingly.